socorro/siglists/README.md

```diff
@@ -0,0 +1,3 @@
+# Signatures Utilities Lists
+
+Great documentation comes here.
```

Comment:
Both reST and Markdown will get rendered in the github ui. We use reST elsewhere. Just mentioning that because I'm not sure why you picked Markdown.

I can't think of whether there's anything that would be in this particular document that would be easier to express with either Markdown or reST, so it's probably not a big deal either way.